Add PHP sample and reorganize sections (#75)
okteto/samples: README.md

> This repository contains examples of how to develop applications directly in your Kubernetes cluster using [okteto](https://github.com/okteto/okteto), with different platforms and deployment technologies.

Reviewer: ```suggestion
This repository contains examples of how to develop applications directly in your Kubernetes cluster using [okteto](https://github.com/okteto/okteto), with different programming languages, frameworks and deployment tool.
```

> ## By languages

Reviewer: ```suggestion
## Programming Languages
```

1. [ASP.NET Core](https://github.com/okteto/aspnetcore-getting-started/)
1. [Golang](https://github.com/okteto/go-getting-started)
1. [Java Gradle](https://github.com/okteto/java-gradle-getting-started)
1. [Java Maven](https://github.com/okteto/java-maven-getting-started)
1. [Node](https://github.com/okteto/node-getting-started)
1. [PHP](https://github.com/okteto/php-getting-started)
1. [Python](python/README.md)

> ## By frameworks

Reviewer: ```suggestion
## Frameworks
```

1. [Django](django/README.md)
1. [Multiple services with Node and React](https://github.com/okteto/movies)
1. [PHP + Webpack](php/README.md)
1. [Ruby on Rails](ruby/README.md)

> ## By deployment tools

Reviewer: ```suggestion
## Deployment Tools
```

1. [Helm](helm/README.md)

> 1. [OpenFaaS](https://medium.com/okteto/how-to-develop-a-serverless-app-with-openfaas-and-okteto-d85435f0eca1)

Reviewer: I'd put this on a 'Serverless' section. While not wrong, I don't think most people relate functions with a deployment tool but rather as a development model.

> ## Remote

Reviewer: ```suggestion
## Remote IDEs
```
